packages/python/pyproject.toml

```diff
@@ -17,7 +17,8 @@ classifiers = [
 dependencies = [
     "grpcio",
     "grpcio-tools",
-    "deprecation"
+    "deprecation",
+    "numpy",
```

Comment:
create another env in the pyproject (stat) similar to the dev one. We don't need to install numpy for armonik